lib/spec/simpleofflinecache.sw-spec.js

```diff
@@ -101,4 +105,40 @@ describe('SimpleOfflineCache instance', function () {
     });
   });
 
+  it('should pass match parameters', function(done) {
+    var cacheName = 'test';
+    var options = {
+      ignoreSearch: true,
+      ignoreMethod: false,
+      ignoreVary: false,
+      prefixMatch: false
+    };
+    var offline = new SimpleOfflineCache(cacheName, options);
+
+    var request = new Request('/index.html?param=zzz');
+
+    var cacheObj = {
+      match: function() {
+        return Promise.resolve(null);
+      },
+      put: function() {
+        return Promise.resolve();
+      }
+    };
+    var spy = sinon.spy(cacheObj, 'match');
+    sinon.stub(self.cacheHelper, 'getCache', function(name) {
+      return Promise.resolve(cacheObj);
+    });
+    offline.onFetch(request).then(function() {
+      expect(self.cacheHelper.getCache.calledOnce).to.be.true;
+      expect(spy.called).to.be.true;
+      expect(spy.calledWith(request.clone(), options)).to.be.true;
+      self.cacheHelper.getCache.restore();
```

Comment:
You need to put this teardown code in the `catch()` _branch_ as well. Maybe you could add a new `describe()` section with its own `beforeEach()` and `afterEach()` but given there is only one test, I'm not blocking on this.